### DISCUSSION TITLE
결정 기간, 책 선정 관련

### DISCUSSION THREAD

**Discussion by hi-hyein** [ThinkAboutSoftware/AcademicConference#2] [General]
1. 결정 기간
4월2일 이 후에 좀 더 정확한 6,7월 일정을 알 수 있을 것 같아서 혹시 결정 기간을  4월 2일까지 연장할 수 있을까요?!
2. 책 선정 관련
책 3권을 다 읽을 수 있을까 하는 부담감이 조금 있는데 2권으로 진행하시는건 어떠신가요? 😢

**Comment by jongfeel** [comment]
이런 제안 좋습니다.

1. 네 괜찮습니다.
4월에 결정해 주시면 4,5,6월 해서 진행하면 가능할 거 같기도 합니다.

2. 책을 다 읽는게 아닐 수 있습니다.
예를 들어 [CODE 코드](http://aladin.kr/p/76Vww)와 같은 책은 600페이지가 넘기 때문에 집중해서 읽는데만도 한달이 넘게 걸릴 수 있습니다.
학술 회의를 한번만 하고 말게 아니기 때문에
멤버들끼리 원하는 책, 원하는 분량 만큼을 선정하고 진행할 예정입니다.
다만 그게 책 한권이 아니라 세권이었으면 좋겠다는 거죠.

책 세권을 두권으로 바꾸지는 않을 것 같은데
모두 다 읽어야 하는건 아니므로 멤버들과 협의해서 진행해 보면 좋을 것 같습니다.
저도 책을 읽는게 너무 힘들어서 일상생활에 지장이 있을 정도로 학술 회의를 진행하고 싶지는 않습니다.

README의 목표(goal)을 다시 알려드리면
- 멘티/멘토간의 친목 도모
- 소프트웨어에 관련된 깊은 대화
- 짧은 여행을 통한 추억 만들기
- 맛있는 음식과 함께하는 호캉스

이렇게 4개인데 1번이 사실 제일 중요합니다. 그래서 처음에 적은 겁니다!

**Reply by hi-hyein** [reply]
조율해주셔서 감사합니다! 👯 👯 
2번의 답변은 이해했어요! 저는 3권 완독이라고 생각을 했는데
답변 주신 내용으로 생각하니 오히려 더 좋네요! 감사합니다ㅎㅎ
